apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -82,7 +82,7 @@ export default class ModerationNotification extends Component {
           showModeratorStatementForm: false,
           alert: {
             type: 'success',
-            message: translated.statementAdded,
+            message: (() => { return (<>{translated.statementAdded} <a href={commentUrl}>{translated.goToDiscussion}</a></>) })(),
```

Comment:
@khamui I couldn't really come up with a nice solution without changing the alert component. Maybe that's too hacky?

Reply:
hehe, well at least it works. I would keep the hacky solution and comment it in code?
later we can adjust the alert to make those kind of things (and maybe other things that is needed) work as well..? If you cool with that, feel free to merge @goapunk

Reply:
btw i think if you use `=> function` you can remove `{}` and `return`

Reply:
I turned it into a real function for now and added a comment